### PR TITLE
Feat/13/task sending page

### PR DESCRIPTION
**Pull request by OdanetsLiliya**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -13,7 +13,16 @@ import { LayoutNavigationComponent } from './layout-navigation/layout-navigation
 import { LayoutPersonalMenuComponent } from './layout-personal-menu/layout-personal-menu.component';
 import { LayoutFooterComponent } from './layout-footer/layout-footer.component';
 import { LayoutLandingPageComponent } from './layout-landing-page/layout-landing-page.component';
-import { MatListModule } from '@angular/material/list';
+import {MatListModule} from '@angular/material/list';
```
Comment (slawiko) [inline src/app/app.module.ts]: Inconsistent spacing

File: src/app/file-apploading/file-apploading.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  selector: 'exc-file-apploading',
+  templateUrl: './file-apploading.component.html',
+  styleUrls: ['./file-apploading.component.scss']
+})
+
+export class FileApploadingComponent implements OnInit {
+  @Output() fileChanged = new EventEmitter<File>();
+  file: File;
+
+  ngOnInit() {
```
Comment (slawiko) [inline src/app/file-apploading/file-apploading.component.ts]: Why do you need this method?

File: src/app/file-apploading/file-apploading.component.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  aploadFile(event) {
+    this.file = event.target.files[0];
+    this.fileChanged.emit(this.file);
+    console.log(event.target.files[0]);
```
Comment (slawiko) [inline src/app/file-apploading/file-apploading.component.ts]: Why two times `event.target.files[0]`

File: src/app/task-sending-page/task-sending-page.component.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.flex{
```
Comment (slawiko) [inline src/app/task-sending-page/task-sending-page.component.scss]: Inconsistent bad indentation and spacing

File: src/app/task-sending-page/task-sending-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  selector: 'exc-task-sending-page',
+  templateUrl: './task-sending-page.component.html',
+  styleUrls: ['./task-sending-page.component.scss']
+})
+export class TaskSendingPageComponent implements OnInit {
+  file: File;
+  dataForPost: DataForPost;
+  constructor(public http: Http) { }
+
+  ngOnInit() {
```
Comment (slawiko) [inline src/app/task-sending-page/task-sending-page.component.ts]: Again

File: src/app/task/task.component.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+p{
+  width: 50vw;
```
Comment (slawiko) [inline src/app/task/task.component.scss]: Why `vw`?

File: src/app/file-apploading/file-apploading.component.html [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<div class='flex'>
+   <input [hidden]='true' id='uploadFile' multiple type="file" (change)='aploadFile($event)'/>
```
Comment (alexey-kozlenkov) [inline src/app/file-apploading/file-apploading.component.html]: It's highly recommended to use double quotes for HTML attributes, but at least keep them consistent
`type="file"` `id='uploadFile'`

Comment (alexey-kozlenkov) [inline src/app/file-apploading/file-apploading.component.html]: _**u**_ploadFile

File: src/app/layout-header/file-apploading.component.spec.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (alexey-kozlenkov) [inline src/app/layout-header/file-apploading.component.spec.ts]: You don't need tests

File: src/app/task-sending-page/task-sending-page.component.html [hunk 9]
```diff
@@ -0,0 +1,11 @@
+<exc-task></exc-task>
+<div class='flex'>
+    <exc-file-apploading (fileChanged)="onFileChanged($event)">
+       fileChanged
```
Comment (alexey-kozlenkov) [inline src/app/task-sending-page/task-sending-page.component.html]: I don't see you're using projection, what is this line for?

File: src/app/file-apploading/file-apploading.component.ts [hunk 10]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Output, EventEmitter, Input } from '@angular/core';
+
+@Component({
+  selector: 'exc-file-apploading',
```
Comment (alexey-kozlenkov) [inline src/app/file-apploading/file-apploading.component.ts]: **uploading**, starts with _u_, single _p_

File: src/app/task-sending-page/task-sending-page.component.ts [hunk 11]
```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import {Http} from '@angular/http';
+
+export interface DataForPost {
```
Comment (alexey-kozlenkov) [inline src/app/task-sending-page/task-sending-page.component.ts]: Interfaces names are nouns, not collocations.
`PostData`, not `DataForPost`.

File: src/app/task-sending-page/task-sending-page.component.scss [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.task {
+  display: flex;
+  flex-direction: column;
+ text-align: center;
+ align-items: center;
+}
+
+.send-button {
```
Comment (slawiko) [inline src/app/task-sending-page/task-sending-page.component.scss]: spacing

Comment (OdanetsLiliya) [reply src/app/task-sending-page/task-sending-page.component.scss]: where???

**Pull request merged**